### ISSUE TITLE
US National Forest and Indian route shields

### ISSUE THREAD

**Issue by claysmalley** [ZeLonewolf/openstreetmap-americana#246]
While we have a [convenient map](https://github.com/ZeLonewolf/openstreetmap-americana/blob/main/doc-img/shield_map_us.svg) to indicate progress on state-level shields, we can't forget about the remaining federal shields. The United States Forest Service and the Bureau of Indian Affairs each signpost their own numbered routes in and around the federal lands they manage.

Relation coverage in both networks is currently incomplete. Tagging seems to have standardized on `network=US:BIA` for Indian Routes, but `network` values for Forest Routes include `US:FS` and `US:NF`. This would need to be standardized before moving forward with implementing shields. Both networks typically have up to 3 digits in their `ref` values, though I wouldn't be surprised if there were ones out there with 4 digits. The Eastern Cherokee Reservation certainly has 4-digit BIA routes, though they aren't signposted with the Indian Route shield (instead, they're next to the street name, as North Carolina's secondary state routes are often signed).

Indian Route shield:

![](https://upload.wikimedia.org/wikipedia/commons/7/75/Indian_Route_591.svg)

Forest Route shield:

![](https://upload.wikimedia.org/wikipedia/commons/d/d6/Forest_Route_33.svg)

**Comment by 1ec5** [comment]
> `network` values for Forest Routes include `US:FS` and `US:NF`. This would need to be standardized before moving forward with implementing shields.

Could some mappers be using these two values to distinguish between [forest primary and secondary routes](https://wiki.openstreetmap.org/wiki/United_States_roads_tagging#National_forest_primary_routes)? Or is this just a slow-motion transition from one tag to the other?

Puerto Rico (#240) also has [_Bosque Nacional_ shields](https://commons.wikimedia.org/wiki/File:Puerto_Rico_Highway_191_-_Signs.jpg) that would look identical to the National Forest shields after simplification.

**Comment by claysmalley** [comment]
Oop. Clearly I needed to do a little more investigation. There are National Forest primary and secondary routes, though current `network` tagging gets even more complicated here.

The guidelines say to include the name of the National Forest in the `network` value, like `US:NFSR:Chequamegon-Nicolet:NF`. I don't think this is necessary, because shields are standardized across all National Forests (with the exception of the Spanish-language text on the Puerto Rico variant).

National Forest secondary route shields can be horizontal or vertical, depending on the level of maintenance of the roadway. The horizontal shields represent well-maintained roads that average vehicles can drive, while vertical shields represent less-maintained roads more suitable for hikers or 4-wheel drive vehicles. FSTopo maps distinguish all three shield designs, but there is currently no tagging scheme to distinguish horizontal from vertical shields.

![](https://wiki.openstreetmap.org/w/images/7/72/US_Forest_Service_Horizontal_Marker.png)

![](https://wiki.openstreetmap.org/w/images/6/6b/US_Forest_Service_Vertical_Marker.png)

**Comment by adamfranco** [comment]
While the shields are standardized across all Forests, the route numbers are only unique within a single Forest.

**Comment by Pengor** [comment]
Just checked US Forest Service Route tagging in Washington and it is definitely rough. I counted 118 roads that I believe would be deserving of `US:NFSR:Forest name:NF` (the primary variety) tagging and found only 1 relation in the whole state (but disjointed and with a typo 😆)...

**Comment by claysmalley** [comment]
Separated BIA routes into a new ticket: #351

**Comment by claysmalley** [comment]
Once #355 is merged, the shield artwork will be available for these routes. The only thing left to do is reach consensus on tagging :grimacing:

**Comment by adamfranco** [comment]
NF shields would be a nice addition to coincide with rendering tracks (#717). 😀 

Example at `map=15.45/43.976043/-73.077052`
![Screen Shot 2023-01-20 at 12 26 31 AM](https://user-images.githubusercontent.com/25242/213623730-7565e1c2-0344-4429-a7c1-e9e35b8e6c30.png)

**Comment by ahallification** [comment]
If anything else, I wonder if we should chunk this into two different issues. 

1. Shields for Forest _Highways_
2. Shields for Forest _Roads_

I'd advocate for doing Forest Highways first, as they're more important and the problem is vastly easier than for Forest Roads.

Forest Highways have only one shape and color across the country, and are predictably maintained. Forest Roads have what appears to be three different shapes, and two different orientations (horizontal and vertical, whereas sometimes the vertical signs have a down arrow in the shape, and other times not). Forest Highways should also always be signed, whereas Forest Roads are more likely to be all over the place. My hope is that by starting small (actually big, the Forest Highways), maybe it could encourage more tagging in OSM and thus, more interest in tackling the bigger challenge of Forest Roads. 

**Tagging**
Having followed OSM Americana and how they've generally decided on tagging schemes for `network=*`, I do think the original guidelines found [here](https://wiki.openstreetmap.org/wiki/United_States_roads_tagging#National_Forest_Road_System) should be tweaked. I'd suggest routes get tagged with `network=US:NFSR:NF` (should this be `FH`? I don't think `NF` is a logical derivative, but changing might require documentation, education, and cleanup) or `network=US:NFSR:FR`. Instead of including each forest name in the `network=*` tag, the forest name should be used in the `operator=` tag at the route level. Sophisticated data consumers would still be able to utilize the new tagging system in the way the used the old tagging system. 

Thoughts? My two takeaways were:
1. Let's just do Forest Highways first, and
2. Should we simplify `network=* `tagging to omit the specific National forest but encourage that data in the `operator=` tag instead.

**Comment by claysmalley** [comment]
Taking the less formal route for a tagging proposal: https://community.openstreetmap.org/t/network-tagging-proposal-for-usfs-roads/97057

**Comment by the-spui-ninja** [comment]
Are there any updates on this? Just mentioning because I found https://www.openstreetmap.org/relation/9443117 while adding Albany County WY route relations and it brought the topic to my mind.

**Comment by ZeLonewolf** [comment]
I'm not clear on whether we have agreement on the tagging for these roads -- if that's settled out, then PR is certainly welcome.
